### ISSUE TITLE
krat.es is not working

### ISSUE THREAD

**Issue by qgustavor** [sumitkolhe/krates#3]
There are some days since [krat.es](https://krat.es/) stopped working. I don't know if the maintainer is aware of it, so I'm posting about it here.

**Comment by sumitkolhe** [comment]
service is back online

**Comment by egpheel** [comment]
I think it's back to not working. Can't do any requests without it timing out

**Comment by Xorbak** [comment]
>  I keep getting rate limit exceeded when trying to post. The last couple of months I also kept getting timeout when trying to change anything

**Comment by sumitkolhe** [comment]
Due to lack of maintainance time I wasn't able to fix the underlying issues. A rewrite of krates is currently work in progress. No ETA as of now.

**Comment by mohithgupta** [comment]
Hi @sumitkolhe, will you continue to maintain this project? It is not working again. I have not checked for a while so not sure when it has been down. Please let me know because I would like to shift my project to another API if you don't. 
Thanks!

**Comment by sumitkolhe** [comment]
@mohithgupta yes i will continue to maintain this project, as stated above, a rewrite is in progress, but no ETA as of now
